README.md

```diff
@@ -2,9 +2,9 @@
 
 ## Introduction
 
-You can use the SpatialOS Unreal GDK Starter Project as the basis for developing a game using the SpatialOS Unreal GDK. To use the Starter Project, you need to follow the [setup guide](https://github.com/improbable/UnrealGDK/blob/master/docs/setup-and-installing.md) for the SpatialOS Unreal GDK. We've enabled use of the `ThirdPersonExampleMap` and a controllable character `StarterGameCharacter` that is based off the Unreal class `Character`.
+You can use the SpatialOS Unreal GDK Starter Project as the basis for developing a game using the SpatialOS Unreal GDK. To use the Starter Project, you need to follow the [setup guide](https://github.com/improbable/UnrealGDK/blob/master/docs/setup-and-installing.md) for the SpatialOS Unreal GDK. We've enabled use of the `ThirdPersonExampleMap` and a controllable character `AStarterGameCharacter` that derives from the Unreal class `ACharacter`.
 
-> Note that the SpatialOS Unreal GDK is currently in **pre-alpha**, and we aren’t supporting any game development on it at this stage.
+> Note that the SpatialOS Unreal GDK is currently in **pre-alpha**, and we do not recommend using it for games in production phase yet.
```

Comment:
Just one thing - I think it should be "in the production phase", i.e. add "the". (Or alternatively just "in production".